### ISSUE TITLE
classNames JS object

### ISSUE THREAD

**Issue by zslabs** [jshjohnson/Choices#40]
Hey!
When trying to customize the `classNames` object during initialization and only impact a single value, it looks like Choices was blowing up a little and not working as expected until I pasted all the default values and customized one of them. Is the intended behavior to need to dupe all the defaults in this area? Thanks!

**Comment by jshjohnson** [comment]
Interesting - do you mind providing your Choices config please? Thanks

**Comment by zslabs** [comment]
Sure, I had the following:

```
const choices = new Choices('.ChoicesSelect select', {
  removeItemButton: true,
  classNames: {
    button: 'Button--reset choices__button'
  }
});
```

**Comment by jshjohnson** [comment]
This should now be resolved in version 2.0.3! Thanks for highlighting this.

**Comment by zslabs** [comment]
Awesome, thanks for the quick-fix!
